### PR TITLE
Add arithmatic2 operations to templateFuncTest

### PR DESCRIPTION
**Pull request by davidsnam-intel**
### Details:
 - *abs, ceiling, erf, floor, log, negative, round, sign, sqrt*
 
### Tickets:
 - *63939*
 - *63945*
 - *63956*
 - *63959*
 - *63967*
 - *63970*
 - *64097*
 - *63990*
 - *63994*

### CODE REVIEW

File: docs/template_plugin/tests/functional/op_reference/abs.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+TEST_P(ReferenceAbsLayerTest, AbsWithHardcodedRefs) {
+    Exec();
+}
+
+}  // namespace
+
+INSTANTIATE_TEST_SUITE_P(
+    smoke_Abs_With_Hardcoded_Refs, ReferenceAbsLayerTest,
+    ::testing::Values(Builder {}
+                          .input({{4}, element::f16, std::vector<ngraph::float16> {1.f, -2.f, 0.f, -4.75f}})
```
Comment (iefode) [inline docs/template_plugin/tests/functional/op_reference/abs.cpp:63]: I think to use template as [here](https://github.com/openvinotoolkit/openvino/blob/204c17cc211c44b9f6977dce4b12a8541cf5d017/docs/template_plugin/tests/functional/op_reference/gather_tree.cpp#L71) is preferable

Comment (davidsnam-intel) [reply docs/template_plugin/tests/functional/op_reference/abs.cpp]: I applied in latest commit

**Comment by ilyachur** [comment]
@davidsnam-intel looks like you had incorrect merge. Please rebase your changes to the latest master

**Comment by davidsnam-intel** [comment]
@ilyachur I will create a new branch. There is some complication arise

**Comment by davidsnam-intel** [comment]
I will close this pull request.
And please see https://github.com/openvinotoolkit/openvino/pull/7843.

**Pull request closed without merging**